pkg/fakeoperator/internal/controllers/db_controller.go

```diff
@@ -58,7 +58,7 @@ func (r *DBReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 		}
 		cond, _ := db.Status.GetCondition(fakev1alpha1.DBReady)
 		if time.Since(cond.LastTransitionTime.Time) < time.Duration(db.Spec.Config.DeletionAfterSeconds)*time.Second {
-			return ctrl.Result{RequeueAfter: time.Second * time.Duration(db.Spec.Config.DeletionAfterSeconds)}, nil
+			return ctrl.Result{RequeueAfter: time.Second*time.Duration(db.Spec.Config.DeletionAfterSeconds) + 10*time.Millisecond}, nil
```

Comment:
why is this changed?

Reply:
I had a problem when I can delete an instance, I thought it's possible when we have `DeletionAfterSeconds=0` and deletion happen in the same second as we change the condition to terminating, but now I see that it's irrelevant.
so I will revert this change